sql/mysql-compatibility.md

```diff
@@ -66,6 +66,10 @@ In TiDB, auto-increment columns are only guaranteed to be incremental and unique
 
 Performance schema tables return empty results in TiDB. TiDB uses a combination of [Prometheus and Grafana](https://pingcap.com/docs/op-guide/monitor/#use-prometheus-and-grafana) for performance metrics instead.
 
+### Query Execution Plan
+
+The output format of Query Execution Plan ( `EXPLAIN` / `EXPLAIN FOR` ) in TiDB is greatly different from that in MySQL. Besides, the output content and the privileges setting of `EXPLAIN FOR` are not the same as those of MySQL. See [Understand the Query Execution Plan](/sql/understanding-the-query-execution-plan.md) for more details.
```

Comment:
This is already mentioned under "Database Administration" in the same document. It originally had its own section for "EXPLAIN", but I moved it under Database administration when the document was getting too long.  We then split this document into Security, Transactions and SQL... If you would like to move it back here you can.

Reply:
Yeah, you're right. I ask kenan for confirmation and he suggested deleting the part in "Data Administration" because the part in "Query Execution Plan" is more specific.

Reply:
No problem - I think because the document split, it is fine to have its own section.  I have changed "Database Administration" to be "Analyze table", because it doesn't make sense to have this heading for only one point any more.